pytorch_lightning/metrics/functional/classification.py

```diff
@@ -138,10 +138,10 @@ def stat_scores_multiple_classes(
         target: torch.Tensor,
         num_classes: Optional[int] = None,
         argmax_dim: int = 1,
+        reduction: str = 'none',
```

Comment:
Can you add a line in docstrings for this?

Reply:
Oh, sure. I missed that.
Do we need updated document (which is on the readthedocs) about this also?

Reply:
no, this is automatically parsed